script.js

```diff
@@ -3,61 +3,61 @@ class Book {
     this.title = title;
     this.author = author;
   }
```

Comment:
- [ ] I think that the implementation of this class can be better if you think about it as the `Books` class rather than the `Book` class. The idea behind a `class` is to gather in one place the behaviors and the properties of something, in this case thinking of adding books and removing books should be related to a list of books and therefore the property here should be the list of books. This makes easier your implementation of the methods and enhancement of the class.

As an advantage, you will avoid repeating the assignment of the books list variable in each of your functions as it is now.

Here is a hint 
```javascript
class Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
   // code to add a new book
    return this.books;
  }

  removeBook(bookID) {
    // code to remove a book
    return this.books;
  }

  getBooks() {
    return this.books;
  }
}
```